### DISCUSSION TITLE
[Support]: Beta Wont Start NGINX

### DISCUSSION THREAD

**Discussion by nosar77** [blakeblackshear/frigate#12178] [Report a Bug]
### Describe the problem you are having

On the same LXC I ran the old frigate, i created a new frigate container using the new documentation and using the old information but It will not run nor go to the UI and im getting these errors. 

### Steps to reproduce

Shutdown old docker

Pull new image, and deploy it using the instructions for docker

### Version

0.14.0-beta3

### Frigate config file

```yaml
mqtt:
  enabled: False

cameras:
  dummy_camera: # <--- this will be changed to your actual camera later
    enabled: False
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:554/rtsp
          roles:
            - detect
```

### Relevant log output

```shell
2024-06-27 19:42:20.995445029  [INFO] Starting NGINX...
[WARN] /sys/fs/cgroup/cpu.max not found. Falling back to /proc/cpuinfo.
./run.user: line 84: tempio: command not found
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
2024-06-27 19:42:21.126572496  [INFO] Service NGINX exited with code 127 (by signal 0)
```

### Operating system

Proxmox

### Install method

Docker CLI

### Network connection

Wired

### Camera make and model

n/A

**Comment by NickM-27** [comment]
See https://github.com/blakeblackshear/frigate/discussions/11755#discussioncomment-9671531

**Reply by nosar77** [reply]
My issue wasn't related to this. But I figured out that the docker command uses the stable image and once I pulled the beta image it worked just fine user error.
